admin/c2cgeoportal_admin/__init__.py

```diff
@@ -15,6 +15,7 @@
 c2cgeoform.default_search_paths = search_paths
 
 _ = TranslationStringFactory('admin')
+client_translation_string_factory = TranslationStringFactory('admin')
```

Comment:
Should be something like "{}-client".format(config.root_package.__name__)

Reply:
`'{}-client'.format(Config.get('package')`